modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -706,13 +706,13 @@ private long streamQuery0(String qry, String schemaName, IgniteDataStreamer stre
                 return rows.size();
             }
         }
-        catch (IgniteCheckedException e) {
-            fail = true;
+        catch (Exception e) {
```

Comment:
why you amend it?

Reply:
I think in case of any exception (unchecked for example), we must set `fail` flag to true.

Reply:
please give one case when we could get not IgniteCheckedException

Reply:
discussed offline: It's better to additionally catch IgniteException (which is unchecked) instead of Exception.